buildout.cfg

```diff
@@ -1,22 +1,19 @@
 [buildout]
-parts = interpreter test
+parts = test
 develop = .
 unzip = true
-index = http://download.zope.org/Zope2/index/2.13.19/
 find-links =
-    http://pypi.python.org/packages/source/P/Products.GenericSetup/
-    http://pypi.python.org/packages/source/P/Products.PluginRegistry/
-    http://pypi.python.org/packages/source/f/five.localsitemanager/
+index = https://pypi.python.org/simple
 extends =
-    http://download.zope.org/Zope2/index/2.13.19/versions.cfg
-
-[interpreter]
-recipe = zc.recipe.egg
-interpreter = python
-eggs =
-    Products.PluggableAuthService
+    http://download.zope.org/Zope2/index/2.13.8/versions.cfg
```

Comment:
What is wrong with `2.13.19`? Maybe it would be even better to use 
https://zopefoundation.github.io/Zope/releases/2.13.28/versions-prod.cfg as `extends` instead of the `index`, as `index` is deprecated.